src/foam/u2/view/RichChoiceView.js

```diff
@@ -392,7 +400,19 @@ foam.CLASS({
                         .start().hide(!! section.hideIfEmpty && resp[index].value <= 0 || ! section.heading)
                           .addClass(self.myClass('heading'))
                           .add(section.heading)
-                        .end()
+                          .end()
+                          .start()
+                            .addClass(self.myClass('selection-view'))
+                            .add('--------------------------')
+                                .callIf(! section.disabled, function() {
+                                  this.on('click', () => {
+                                    self.fullObject_ = undefined;
+                                    self.data = 0;
+                                    self.isOpen_ = false;
+                                    self.cross=true;
```

Comment:
Please remove.

Reply:
we need to close the close the option list